packages/webpack/src/runtime/runtimeUnlocked.js

```diff
@@ -0,0 +1,47 @@
+/// <reference path="./lavamoat.d.ts" />
+/* global LAVAMOAT */
+
+const { create, freeze, defineProperty } = Object
+const warn = typeof console === 'object' ? console.warn : () => {}
+warn('LavaMoat: using unlocked runtime')
+
+const { NAME_globalThis, NAME_scopeTerminator, NAME_runtimeHandler } =
+  LAVAMOAT.ENUM
+
+/**
+ * Wraps the webpack runtime with Lavamoat security features.
+ *
+ * @param {string} resourceId - The identifier of the resource.
+ * @param {any} runtimeKit - The runtime kit containing bits from the webpack
+ *   runtime.
+ * @returns {Object} - An object containing the wrapped runtime and other
+ *   related properties.
+ */
+const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
+  let { module } = runtimeKit
+  const runtimeHandler = runtimeKit
+
+  // allow setting, but ignore value for /* module decorator */ module = __webpack_require__.nmd(module)
+  defineProperty(runtimeHandler, 'module', {
+    get: () => module,
+    set: () => {},
```

Comment:
is the intent to provide a no-op, or is it to provide a read-only prop?

Reply:
the intent is to avoid throwing errors on attempts of breaking module.